core/mod/ALU.mod

```diff
@@ -135,7 +135,11 @@ VERBATIM {
     INFOCAST;
     Info* info = (Info*)hoc_Emalloc(sizeof(Info)); hoc_malchk();
     info->psize_ = 10;
+#ifdef NRN_MECHANISM_DATA_IS_SOA
```

Comment:
Oh! just additional question: I am not up to date with current usage/recommendation feom Olli: is it better/possible to use NRN_VERSION (>=9) macros than NRN_MECHANISM_DATA_IS_SOA? Later seems bit low level and shoudln’t be exposed in user MOD files if possible ?

(I see that it exists in old code but I wonder if it’s required)

Reply:
> Personally, I like NRN_MECHANISM_DATA_IS_SOA because it's more expressive than NRN_VERSIO

As a developer, yes!  As end users/scientists, not at all :) So I believe eversion macros are better.